packages/serverpod/lib/src/protocol/database/database_definition.yaml

```diff
@@ -16,3 +16,6 @@ fields:
   ### Modules installed in the database, together with their version. Only
   ### set if known.
   installedModules: List<DatabaseMigrationVersion>?
+
+  ### The version of the database definition.
+  version: int 
```

Comment:
Suggestion: migrationApiVersion or something similar could be a more clear name as it may not otherwise be clear which version this refers to.